src/frontend/komponenter/Fagsak/Vilkårsvurdering/VilkårTabell/VilkårTabell.tsx

```diff
@@ -86,7 +86,7 @@ const GeneriskVilkår: React.FC<IProps> = ({
     };
 
     return (
-        <div className={'generisk-vilkår'}>
+        <div className={'vilkår-tabell'}>
```

Comment:
Husker ikke helt i farta, men er dette en tabell eller ligner det bare? Kan være litt rart om det kodes som en tabell, men ikke er det :) Sånn UU-messig.

Reply:
Tenker du på at tabellen ligger inne i en div og burde vært på toppnivå for at klassen skal kalles tabell? :)